src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -524,24 +640,35 @@ public function run() {
 		wp_update_plugins(); // Check for plugin updates.
 		$plugin_updates = get_site_transient( 'update_plugins' );
 		if ( $plugin_updates && ! empty( $plugin_updates->response ) ) {
+			error_log( '  Automatic plugin updates starting...' );
```

Comment:
Same here: do we need to log everything that happens during the update process?

Reply:
Same as above, just for helping testers. :slightly_smiling_face:

Reply:
Yeah I just saw this in the Make/Core blogpost 👍